src/index.html

```diff
@@ -7,6 +7,345 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="page-header">
+      <img src="./images/logo.png" alt="MOYO logo">
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="" class="nav__link is-active">
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="" class="nav__link">Samsung</a>
```

Comment:
```suggestion
            <a href="#" class="nav__link">Samsung</a>
```
Не оставляй href пустым